### DISCUSSION TITLE
Maximum number of users allows to eXo

### DISCUSSION THREAD

**Discussion by Rajamuthukrishna** [exoplatform/exo-community#77] [Q&A]
Hello Team,

Kindly share the maximum number of users can allow in eXo application.

Regards,
Raja

**Comment by rdenarie** [comment]
Hello @Rajamuthukrishna 
It depends a lot of the usage of the platform, of the server which runs exoplatform, and lot of parameters. It is difficult to bring an answer to this question.

For the next time, can you ask you question on community.exoplatform.com ?
You have different spaces in which you can ask questions, and you will have more visibility.

Thanks
